### DISCUSSION TITLE
Support for headings in tabs?

### DISCUSSION THREAD

**Discussion by pideu-mh** [squidfunk/mkdocs-material#6217] [Q&A] [answered]
I am using content tabs which are working great. I can put a lot of content into them and they just work. With one exception, where things only work partially: headings.

It is possible to put headings in tabs and everything renders correctly. However, the downside of this is that this does not reflect correctly in the table of contents section on the right of the page.

For example, if you have:
```
## Section 1

=== "A"
    ## Section 2a
    In section 2a
=== "B"
    ## Section 2b
    In section 2b

## Section 3
```
then the table of contents shows:
![image](https://github.com/squidfunk/mkdocs-material/assets/28350507/56b12a42-089c-4e3e-8234-5e4260c67afb)
I naively assumed that the table of contents changes with the tab that is selected, too, ie. when tab "A" is selected, then the table of contents would show 1, 2a, 3 and if tab "B" is selected, then it would show 1, 2b, 3.

I can totally understand if this is impossible to achieve, because the table of contents may be statically rendered from the contents of the page and that contains both sections 2a and 2b.

Or is this something that should work in theory, but currently does not work in practice?

**Comment by squidfunk** [comment, marked as answer]
This was already asked before:

- #4828 
- #3376

This is why using the search is always a great idea. Took me 1m 😉

**Comment by pideu-mh** [comment]
Thanks for the quick response. In my defense I have to say that I searched for combinations of "headings tabs" which did not reveal the questions you quoted, but I did not try "headers" instead, that would have found them.

My bad, I'm sorry for the duplicate request, @squidfunk.
